daffodil-propgen/src/main/scala/org/apache/daffodil/propGen/TunableGenerator.scala

```diff
@@ -352,12 +352,12 @@ class TunableEnumDefinition(
 """.trim.stripMargin
 
   private val scalaEnums = {
-    val scalaEnumValues = allEnumerationValues.map { e => e.head.toUpper + e.tail }
+    val scalaEnumValues = allEnumerationValues.map { e => s"${e.head.toUpper}${e.tail}" }
```

Comment:
If you are just concatenating strings then using "+" is clearer than the string interpolation which I must read very carefully to see if a whitespace is in there, or some other small characters.

Reply:
Char + is conctatenation to a string is deprecated in 2.13, and string interpolation was the recommended fix from Intellij, we can either do that or convert the char to a string before using +. Though I prefer string interpolation personally

https://docs.scala-lang.org/scala3/guides/migration/incompat-dropped-features.html#any2stringadd-conversion

Reply:
Gaaak.  A String is a `Seq[Char]`. Appending `T` to `Seq[T]` at the front of the sequence is one of the most basic operations in functional programming. It is `cons` from Lisp. 

Perhaps `+` is the wrong operator here? Maybe these should be the actual `cons` operator i.e., is that `char :: string` or `char +: string` ?

Reply:
But to your point sticking with string concatenation will look something like

e/head.toUpper.toString + e.tail

Is that preferable?

Reply:
Calling toString is not, But `e.head.toUpper +: e.tail` is preferable to the string interpolation.